runners/genlayer-py-std/src/genlayer/__init__.py

```
@@ -80,26 +80,30 @@
 	'u96',
 )
 
+import os
+
 from .py.types import *
 from .py.storage import *
 
-if not typing.TYPE_CHECKING:
+_gen_docs = os.getenv('GENERATING_DOCS', 'false') == 'true'
+
+if not typing.TYPE_CHECKING and not _gen_docs:
```

Ruff (0.11.9)

`typing` may be undefined, or defined from star imports

(F405)